cwms-data-api/src/main/java/cwms/cda/api/OfficeController.java

```diff
@@ -121,17 +122,23 @@ public void getAll(Context ctx) {
                     + " want more information for"),
             queryParams = @OpenApiParam(name = FORMAT,
                     deprecated = true,
-                    description = "(Deprecated in favor of Accept header) Specifies the encoding "
-                            + "format of the response. Valid value for the format field for this "
-                            + "URI are:\r\n1. tab\r\n2. csv\r\n 3. xml\r\n4. json (default)"
+                    description = "(CWMS-Data-Format-Deprecated: 2024-11-01 in favor of Accept header)"
+                            + " Specifies the encoding format of the response. Valid value for the format "
+                            + "field for this URI are:\r\n"
+                                + "1. `tab`\r\n"
+                                + "2. `csv`\r\n"
+                                + "3. `xml`\r\n"
+                                + "4. `json` (default)"
             ),
```

Comment:
Remove the numbered list format here and make it a bullet list like in getAll.   I thought I replaced that everywhere but I guess I missed this one.

Reply:
Updated to bullet points, I'll push once you're done reviewing